src/utils/misc-utils.ts

```diff
@@ -1,5 +1,6 @@
 export const notUndefined = <T>(x: T | undefined): x is T => x !== undefined;
 export const delay = (timeout?: number) => new Promise(r => setTimeout(r, timeout));
+export const parseBooleanFromURLParam = (bool: string | undefined) => bool !== "false" && bool !== undefined;
```

Comment:
would it not be better to have `=== "true"`?

Reply:
I was thinking about this.. sometimes "?recursive" is a valid way of saying true afaik

Reply:
so I guess I could do "true" or "" ?